src/WebManager/Services/DockerApiService.cs

```diff
@@ -106,7 +108,7 @@ public DockerApiService()
         /// <summary>
         /// Reads a manifest.
         /// </summary>
-        public async Task<Tuple<string, HttpStatusCode>> Manifest(RegistryCredential cred, string repo, string tag)
+        public async Task<Tuple<string, HttpStatusCode,string>> Manifest(RegistryCredential cred, string repo, string tag)
```

Comment:
nit: space between `,` and `string`.

On a second thought, replace the use of Tuple by a strongly-typed object with the required properties.